manifests/params.pp

```diff
@@ -1,19 +1,34 @@
 # == Class: contrail::params
 #
 class contrail::params(
-  $analytics         = {},
-  $analyticsdatabase = {},
-  $config            = {},
-  $control           = {},
-  $database          = {},
-  $webui             = {},
-){
-
+  $analytics                  = {},
```

Comment:
yes, container_url and container_tag are exposed in my clone https://github.com/alexey-mr/contrail-tripleo-heat-templates/tree/stable/newton.  Let me know if there is something else to be exposed.